### ISSUE TITLE
Broken in gulp-mocha 4.3.1

### ISSUE THREAD

**Issue by soundstep** [ilearnio/module-alias#11]
Just reporting.
Aliases are working with gulp-mocha@3.0.1, but not with gulp-mocha@4.3.1.
I will try to investigate when I get time.

```
const path = require('path');
const alias = require('module-alias');

alias.addAliases({
    'app': path.resolve(__dirname, '../')
});
```

```const path = require('path');
const alias = require('module-alias');

alias.addAliases({
    'root': path.resolve(__dirname, '../../')
    'tests': path.resolve(__dirname, '../../tests')
});
```

**Comment by soundstep** [comment]
Some more info.
The problem is not coming from mocha itself but gulp-mocha.
I supposed that the problem appeared as this new version of gulp-mocha is using execa:
https://www.npmjs.com/package/execa

**Comment by wilsonespina** [comment]
@soundstep Adding an additional `NODE_PATH = '.'` in the gulp tasks has solved this issue:
```
process.env.NODE_PATH = '.';
```
With the fix, Aliases are working with gulp-mocha@6.0.0
